testdata/1494538317_add-phone-number-to-users.up.sql

```diff
@@ -0,0 +1 @@
+alter table users add column phone text;
```

Comment:
Tanel, can you please use underscore instead on dashes in file names of test files, otherwise it looks weird, but you'll need to change it in tests as well.